### DISCUSSION TITLE
Add more properties to control Terminal via status effects

### DISCUSSION THREAD

**Discussion by SomeRandomNoobKekeke** [FakeFishGames/Barotrauma#14451] [Ideas and suggestions]
E.g. There is 
https://github.com/FakeFishGames/Barotrauma/blob/0aee53976202c7ea5c0bb3f26669a97c8dc2400c/Barotrauma/BarotraumaShared/SharedSource/Items/Components/Signal/Terminal.cs#L56-L59
But no way to clear terminal
The only workaround is to push 60 (private const int MaxMessages = 60; lol) empty messages

### It would be much easier to make ingame docs for mods if
There was a "Clear" property

MaxMessages would be a property instead of a const, you could set it to one, ShowMessage with buttons from CustomInterface and pretend these are pages

There was mb a property to delete n'th message

StatusEffect spawnitem with equip="true" could equip items to hands, then you could swap books with different button layouts in players hands without closing it

**Comment by BlackMoons** [comment]
This would be great, More mods should really have functional ingame documentation and iv come up against this limitation myself recently.
